### PR TITLE
cnap/core: add the implementation for inference queue client

### LINKED ISSUE #41: design inference queue client for micro services

### PR DESCRIPTION
**Pull request by Hulongyin**
This PR is to fix #41 , design and implement inference queue client for micro services.

### CODE REVIEW

File: cnap/core/inferqueue.py [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+            LOG.info("Unregister inference queue for topic: %s", topic)
+            self._conn.delete(key)
+            self._conn.delete(topic)
+
+class KafkaInferQueueClient(InferQueueClientBase):
+    """
+    Kafka implementation for inference queue client.
+    """
+
+    MAX_RECONNECTION_TIMES = 5
```
Comment (leyao-daily) [inline cnap/core/inferqueue.py]: Duplicate definition, may be we declare it in base class?

Comment (Hulongyin) [reply cnap/core/inferqueue.py]: Thanks, fixed.

File: cnap/requirements.txt [hunk 2]
```diff
@@ -2,3 +2,5 @@ redis>=4.3.0
 protobuf
 numpy
 opencv-python
```
Comment (leyao-daily) [inline cnap/requirements.txt:3]: please make it sorted.

Comment (Hulongyin) [reply cnap/requirements.txt:3]: Thanks, fixed.

**Review by leyao-daily — APPROVED** [review]
LGTM.

**Comment by Hulongyin** [comment]
Closes #41

**Pull request merged**